src/components/smart-overflow.cjsx

```diff
@@ -52,7 +53,7 @@ SmartOverflow = React.createClass
   render: ->
     {className} = @props
 
-    classes = "#{className} smart-overflow"
+    classes = classnames 'openstax-smart-overflow', className
 
     <div className={classes} style={@state.style}>
       {@props.children}
```

Comment:
I see what you mean, but i feel like that decreases readability, especially if for whatever reason we need to use more classes or logic that may overflow the recommended line-length